src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
     )
     create_workdir: Optional[bool] = Field(
         True,
-        description="Create base and/or user-specific working directories at runtime.",
-        title="Create Workdir",
+        description='Create base and/or user-specific working directories at runtime.',
+        title='Create Workdir',
     )
-    use_templates: Optional[bool] = Field(
-        True,
-        description="Use the templated versions of the scripts and write them to the working directory.",
-        title="Use Templates",
+    use_local_scratch: Optional[bool] = Field(
+        False, description='To use local storage as working dir for jobs', title='Use Local Scratch'
     )
+    local_scratch_dir: Optional[str] = Field(
+        None,
+        description='Path to the local scratch directory to be used as jobs working dir',
+        title='Local Scratch Dir',
+    )
+
+
+class AnalyzeResponse(BaseModel):
+    analytics: Optional[Analytics] = None
+    evaluators: List[AnalyzeResponseEvaluator] = Field(..., title='Evaluators')
+    scalers: List[AnalyzeResponseScaler] = Field(..., title='Scalers')
 
 
 class ComputeResourceSet(BaseModel):
     name: Optional[str] = Field(
-        "default", description="Name of the compute resource set.", title="Name"
+        'default', description='Name of the compute resource set.', title='Name'
     )
-    id: Optional[str] = Field(None, description="ID for this set.", title="Id")
+    id: Optional[str] = Field(None, description='ID for this set.', title='Id')
     scaler_id: Optional[str] = Field(
         None,
-        description="Temporary. To be removed after transitioning to client_id.",
-        title="Scaler Id",
+        description='Temporary. To be removed after transitioning to client_id.',
+        title='Scaler Id',
     )
     last_modified: Optional[datetime] = Field(
-        None, description="Last modified time.", title="Last Modified"
+        None, description='Last modified time.', title='Last Modified'
     )
     backend: Optional[
-        Union[KubernetesKedaBackend, OrchestrationInterfacesBackend, LocalBackend, MockupBackend]
+        Union[
+            KubernetesKedaBackend,
+            OrchestrationInterfacesBackend,
+            OCMBackend,
+            LocalBackend,
+            MockupBackend,
+        ]
     ] = Field(
-        {"debug": False, "plugin_name": "local"},
-        description="Backend to use in this compute resource set.",
-        discriminator="plugin_name",
-        title="Backend",
-    )
-    scaling_strategy: Optional[
-        Union[MaxAvailableResourceScaling, KubernetesResourceScaling]
-    ] = Field(
-        {"match_all_requirements": False, "plugin_name": "max_available_resource_scaling"},
-        description="Scaling strategy to use in this compute resource set.",
-        discriminator="plugin_name",
-        title="Scaling Strategy",
+        {'debug': False, 'plugin_name': 'local'},
+        description='Backend to use in this compute resource set.',
```

Comment:
```suggestion
        description='Backend to use in the compute resource set.',
```
We prefer using "the" over "this" where possible.